.gitignore

```diff
@@ -20,7 +20,8 @@ cordova/bundle.keystore
 .DS_store
 
 # artefacts for Visual Studio Code
-/.vscode/
+.vscode/*
+!.vscode/extensions.json
```

Comment:
@McGiverGim Adding this so that we can recommend some extensions as well as having the commit hooks.

Reply:
Ok, I'm not too sure if this will work if the user has its own file.